docs/sources/operators-guide/deploy-grafana-mimir/upgrade-helm-chart-2.1-to-3.0/_index.md

```diff
@@ -125,6 +127,13 @@ Several parameters that were available in version 2.1 of the mimir-distributed H
        replicas: 3
      ```
 
+1. Decide whether or not you need to update the Mimir configuration due to memcached changes:
+
+   - If you have copied and customized `mimir.config`, you'll need to merge in the changes from the chart as the settings for memcached have changed.
+   - If you do not customize `mimir.config`:
+     - In general the defaults should work so there is no need to customise memcached related parameters via `mimir.structuredConfig`.
+     - If you have set the memcached related Mimir parameters (`*.memcached.addresses` or `*.memcached.max_item_size`) in `mimir.structuredConfig`, then those need to be updated. Please consult the `mimir.config` for defaults.
```

Comment:
I would move it to be on the same level as the instructions about `mimir.config`

```suggestion
    - If you have set the memcached-related Mimir parameters (`*.memcached.addresses` or `*.memcached.max_item_size`) in `mimir.structuredConfig`, then those need to be updated. Please consult the `mimir.config` for defaults.
```